.openhands/microagents/repo.md

```diff
@@ -13,7 +13,7 @@ You don't need to do this unless the user asks you to, or if you're trying to ru
 Before pushing any changes, you should ensure that any lint errors or simple test errors have been fixed.
 
 * If you've made changes to the backend, you should run `pre-commit run --all-files --config ./dev_config/python/.pre-commit-config.yaml`
-* If you've made changes to the frontend, you should run `cd frontend && npm run lint:fix && npm run build ; cd ..`
+* If you've made changes to the frontend, you should run `cd frontend && npm run lint:fix && npm run build ; cd ..` and make sure `make build-frontend` works locally
```

Comment:
Why are we adding this in addition to `npm run build`? Seems like it'll end up running it twice

Reply:
🤔 ughhh you are probably right.. this is probably redundant